java-checks/src/main/java/org/sonar/java/filters/LombokFilter.java

```diff
@@ -118,7 +118,7 @@
   public void visitImport(ImportTree tree) {
     String fullyQualifiedName = ExpressionsHelper.concatenate((ExpressionTree) tree.qualifiedIdentifier());
 
-    excludeLinesIfTrue(fullyQualifiedName.startsWith("lombok."), tree, UselessImportCheck.class);
+    excludeLinesIfTrue("lombok.var".equals(fullyQualifiedName) || LOMBOK_VAL.equals(fullyQualifiedName), tree, UselessImportCheck.class);
```

Comment:
Strange that `fullyQualifiedName` once is compared with literal and once with constant...